consensus/engine/consensus_engine.go

```diff
@@ -110,11 +110,12 @@ type Engine interface {
 	// and assembles the final block.
 	// Note: The block header and state database might be updated to reflect any
 	// consensus rules that happen at finalization (e.g. block rewards).
+	// sigsReady signal indicates whether the commit sigs are populated in the header object.
```

Comment:
Do you think adding a note in comment saying that the `Finalize` is blocked would be better?

Reply:
Done.